client/i18n/locales/english/translations.json

```
@@ -450,7 +450,8 @@
       "link-li-3": "If you do not have a transcript link, click the \"Create link\" button to create one.",
       "link-li-4": "Click the \"Copy link\" button to copy the transcript URL.",
       "link-li-5": "Paste the URL into the input below and click \"Link Account\".",
-      "transcript-label": "Your Microsoft Transcript Link"
+      "transcript-label": "Your Microsoft Transcript Link",
+      "invalid-transcript": "Your transcript link is not correct, it should have the following form: https://learn.microsoft.com/<LOCALE>/users/<USERNAME>/transcript/<ID>"
```

```suggestion
      "invalid-transcript": "Your transcript link is not correct, it should have the following form: https://learn.microsoft.com/LOCALE/users/USERNAME/transcript/ID - check the UPPERCASE items as per your link are correct."
```

Slightly more fool-proof, and while there is it possible to wrap that link and the `UPPERCASE` in a code tag or something, so people dont translate it?

---

I went for 'in' over 'as per' just because the latter is a little idiosyncratic.

As for the code tags, yeah, should be possible. I'll try and wrangle a `Trans` component and get that working.